File: src/05-ciclos/mientras/mientras.pas
Program Mientras
Var
   numero, resultado, potencia: Integer;

Begin
   Write('Introduzca un numero: ');
   ReadLn(numero);

   Write('Introduzca una potencia: ');
   ReadLn(potencia);

   resultado := 1;

   While potencia > 0 Do
   Begin
      resultado := resultado * numero;
      potencia := potencia - 1;
   End;

   WriteLn(resultado);
End.

File: src/05-ciclos/mientras/mientras.py
if __name__ == "__main__":
    resultado = 1
    numero = int(input("Introduzca un numero: "))
    potencia = int(input("Introduzca una potencia: "))

    while (potencia > 0):
        resultado = numero * resultado
        potencia = potencia - 1

    print(resultado)
